adodb-datadict.inc.php

```diff
@@ -520,7 +520,7 @@ function RenameColumnSQL($tabname,$oldcolumn,$newcolumn,$flds='')
 			list($lines,$pkey,$idxs) = $this->_GenFields($flds);
 			// genfields can return FALSE at times
 			if ($lines == null) $lines = array();
-			list(,$first) = each($lines);
+			$first  = current($lines);
```

Comment:
This is not directly related to your change, but the *list()* call on the next line will throw a PHP notice if *preg_split()* returns an array with a single element (which can happen if $first is empty or does not contain any whitespace)